apps/judicial-system/api/src/app/modules/defendant/models/defendant.model.ts

```diff
@@ -126,4 +126,10 @@ export class Defendant {
 
   @Field(() => String, { nullable: true })
   readonly alternativeServiceDescription?: string
+
+  @Field(() => String, { nullable: true })
+  readonly verdictForDefendant?: string
```

Comment:
Please use the field `ruling` in the case model instead as per discussion with Guðný.